il.cpp

```diff
@@ -668,13 +668,39 @@ bool GetLowLevelILForInstruction(Architecture* arch, const uint64_t addr, LowLev
 
 	case XED_ICLASS_AND_LOCK: // TODO: Add Lock construct
 	case XED_ICLASS_AND:
+		il.AddInstruction(
+			WriteILOperand(il, xedd, addr, 0, 0,
+				il.And(opOneLen,
+					ReadILOperand(il, xedd, addr, 0, 0),
+					ReadILOperand(il, xedd, addr, 1, 1),
+					IL_FLAGWRITE_ALL)));
+		// Setting parity flag.
```

Comment:
There is no need to explicitly set the PF here. It is automatically handled by the flag system.

Reply:
It doesn't seem so on my side after testing:
![image](https://user-images.githubusercontent.com/7302683/150865499-08a5a8b1-89e4-4044-972a-bb31eacfb645.png)
Tested with this listing:
```
test_and_pf PROC
	and rcx, rdx
	setp al
	ret
test_and_pf ENDP
```

Reply:
Ah right, I remember, I already removed this code in the last commit of this PR as it was wrong anyway. So I shouldn't need to change anything here :)